gnomad/resources/resource_utils.py

```diff
@@ -109,16 +109,23 @@ class TableResource(BaseResource):
 
     expected_file_extensions: List[str] = [".ht"]
 
-    def ht(self, force_import: bool = False) -> hl.Table:
+    def ht(
+        self,
+        force_import: bool = False,
+        read_args: Optional[Dict[str, Any]] = None,
+    ) -> hl.Table:
         """
         Read and return the Hail Table resource.
 
+        :param force_import: If ``True``, force the import of the resource even if it
+            already exists.
+        :param read_args: Any additional arguments to pass to hl.read_table.
         :return: Hail Table resource
         """
         if self.path is None or force_import:
             return self.import_func(**self.import_args)
         else:
-            return hl.read_table(self.path)
+            return hl.read_table(self.path, **(read_args or {}))
```

Comment:
another basic question: if `read_args` defaults to `{}` instead of None, then you wouldn't need the `(read_args or {})`. did you set this to None just to keep the argument Optional?

Reply:
Yeah, I was really just following what was done above for `import_args: Optional[Dict[str, Any]] = None,`

I'm fine with changing to default to {} if you prefer

Reply:
ah OK, works for me!